packages/web/src/common/store/ui/add-to-playlist/actions.ts

```diff
@@ -8,7 +8,9 @@ export const CLOSE = 'ADD_TO_PLAYLIST/CLOSE'
 
 export const requestOpen = createCustomAction(
   REQUEST_OPEN,
-  (trackId: ID, trackTitle: string) => ({ trackId, trackTitle })
+  (trackId: ID, trackTitle: string) => {
```

Comment:
linter or do you like explicit returns? lol
If so should change the one below for consistency

Reply:
I think I added a console and then didn't clean up, updating :)